### ISSUE TITLE
Change the spacing in import statements

### ISSUE THREAD

**Issue by xabikos** [xabikos/vscode-javascript#17]
As requested [here](https://github.com/xabikos/vscode-react/issues/18) I am thinking on changing the import statements way from `import React, {Component} from 'react';` to `import React, { Component } from 'react';` which includes an additional space when importing only a part of the module. 
This is aligned with the [airbnb styleguide](https://github.com/airbnb/javascript#modules) 

As a lot of people have been downloaded the snippets and the issue has not been raised so far I am a bit skeptical to go for this change.

**Comment by gluons** [comment]
So `ima` will change, right? 😃

```javascript
import {${2:originalName} as ${3:alias} } from '${1:module}';$0
```
⬇️
```javascript
import { ${2:originalName} as ${3:alias} } from '${1:module}';$0
```

**Comment by xabikos** [comment]
Fixed with fafc52df00081db1623588c1ac60d0b3496450a9
